Dockerfile

```diff
@@ -134,3 +134,7 @@ ENTRYPOINT ["nodeplugin"]
 FROM alpine:3.18 AS controller
 COPY --from=controller-builder /app/disperser/bin/controller /usr/local/bin
 ENTRYPOINT ["controller"]
+
+FROM alpine:3.18 AS relay
+COPY --from=common-builder /app/relay /usr/local/bin
+ENTRYPOINT ["relay"]
```

Comment:
We also need to add the build stage for relay, right?

Reply:
Thx